src/app/beer_garden/api/stomp/server.py

```diff
@@ -125,34 +126,44 @@ def __init__(
         if subscribe_destination:
             self.conn.set_listener("", OperationListener(self.conn, send_destination))
 
-    def connect(self, connected_message=None):
+    def connect(self, connected_message=None, gardens=None):
         wait_time = 0.1
-        while not self.conn.is_connected() and self.bg_active:
-            try:
-                self.conn.connect(
-                    username=self.username,
-                    passcode=self.password,
-                    wait=True,
-                    headers={"client-id": self.username},
-                )
-                if self.subscribe_destination:
-                    self.conn.subscribe(
-                        destination=self.subscribe_destination,
-                        id=self.username,
-                        ack="auto",
-                        headers={
-                            "subscription-type": "MULTICAST",
-                            "durable-subscription-name": self.subscribe_destination,
-                        },
+        if (
+            self.host_and_port[0][0]
+            and self.host_and_port[0][1]
```

Comment:
Can't `host_and_port[0]` be `None`? If that happens this is going to blow up.

Reply:
@hazmat345 `host_and_port` defaults to `None` if nothing is passed in, but a value is always passed in. If the value of `host` and `port` are `None` then value passed in on init is ```host_and_port = [(None, None)]```, so `host_and_port` should not be `None`.